.vscode/launch.json

```diff
@@ -60,6 +60,20 @@
             "device": "MIMXRT1062xxx6A",
             "svdFile": "${workspaceFolder}/.vscode/MIMXRT1062.svd",
             "preLaunchTask": "Build NXP mimxrt1060"
+        },
+        {
+            "name": "NXP mimxrt1060 ADU: Local J-Link",
+            "type": "cortex-debug",
+            "request": "launch",
+            "servertype":"jlink",
+            "postStartSessionCommands": ["monitor reset"],
+            // TODO: Currently debuger fails with runToMain true.
```

Comment:
Please make sure that all `TODO` are formatted as `// TODO: #nnn` where #nnn is a GitHub issue tracking the solution.

Reply:
Created: https://github.com/Azure-Samples/iot-middleware-freertos-samples/issues/357